RecoMTD/TrackExtender/python/trackExtenderWithMTD_cfi.py

```diff
@@ -0,0 +1,31 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoMTD.TrackExtender.PropagatorWithMaterialForMTD_cfi import *
+
+_mtdRecHitBuilder = cms.string('MTDRecHitBuilder')
+_mtdPropagator = cms.string('PropagatorWithMaterialForMTD')
+
+trackExtenderWithMTD = cms.EDProducer(
+    'TrackExtenderWithMTD',
+    tracksSrc = cms.InputTag("generalTracks"),
```

Comment:
any chance to get a fillDescriptions for this one as well, similar to the other producer modules?
Please clarify.
Thanks.